idl/storage_service.idl.hh

```diff
@@ -16,7 +16,6 @@ struct raft_topology_cmd {
         barrier,
         barrier_and_drain,
         stream_ranges,
-        fence_old_reads,
```

Comment:
Can't be removed, if we maintain compatibility.

Reply:
It's not published yet, under experimental flag.

Reply:
We do not at this point.